examples/gce-acme/startup.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash -xe
```

Comment:
Bash scripts should also have the Google header. If you'd like you can use our license verifier script: https://github.com/terraform-google-modules/terraform-google-project-factory/tree/master/test

Reply:
Thanks, I wasn't able to get this to work for a license header in my openapi_spec.yaml file, any suggestions?

Reply:
We have a license on a yaml file here: https://github.com/terraform-google-modules/terraform-google-cloud-datastore/blob/master/example/yaml/index.yaml

That repo has the appropriate boilerplate and code for testing yaml headers.

Reply:
Ok, I'll add the header but I still couldn't get the `verify_boilerplate.py` script to pass.

Reply:
Alright, I can take a look at that later on - fine to merge without it.